src/main/resources/citizen/templates/document/defendantResponseReceipt.html

```diff
@@ -985,122 +1003,155 @@ <h2 class="heading-medium">
   </thead>
   <tbody>
 
-  <tr>
-    <td>
-      <div>
-        Support required for a hearing
-      </div>
-    </td>
-    <td>
-      <div>
-        {% for support in hearingContent.supportRequired %}
-        {{ support }}<br/>
-        {% endfor %}
-      </div>
-    </td>
-  </tr>
+  {% if hearingContent.supportRequired is not empty %}
+    <tr>
+      <td>
+        <div>
+          Support required for a hearing
+        </div>
+      </td>
+      <td>
+        <div>
+          {% for support in hearingContent.supportRequired %}
+          {{ support }}<br/>
+          {% endfor %}
+        </div>
+      </td>
+    </tr>
+  {% endif %}
 
-  <tr>
-    <td><b>Hearing location</b></td>
-  </tr>
-  <tr>
-    <td>
-      Preferred hearing centre
-    </td>
-    <td>
-      <div>{{ hearingContent.hearingLocation }}</div>
-    </td>
-  </tr>
-  <tr>
-    <td>
-      <div>
-        Briefly explain your reasons
-      </div>
-    </td>
-    <td>
-      <div>
-        {{ hearingContent.locationReason }}
-      </div>
-    </td>
-  </tr>
+  {% if hearingContent.hearingLocation is not empty %}
+    <tr>
+      <td><b>Hearing location</b></td>
```

Comment:
I guess this should always appear as this is the default question to everyone

Reply:
i think Company/Org doesn't choose court, so this maybe empty